packages/react-devtools-shared/src/hooks/__tests__/updateMockSourceMaps.js

```diff
@@ -323,7 +323,11 @@ async function bundle() {
     plugins: [
       rollupResolve(),
       commonjs(),
-      babel({presets: ['@babel/preset-react'], sourceMap: true}),
+      babel({
+        babelHelpers: 'bundled',
```

Comment:
- https://github.com/rollup/plugins/blob/master/packages/babel/CHANGELOG.md#500
  - > Removed `externalHelpers` & `runtimeHelpers` options. There is now a single `babelHelpers` option which can take one of `'bundled'`, `'inline'`, `'runtime'` and `'external'` as a value. The default is `'bundled'` which matches `4.x` behavior, **but it is recommended to configure this option explicitly.**
- https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers